integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/nextmove/message/BugFix610.java

```diff
@@ -0,0 +1,43 @@
+package no.difi.meldingsutveksling.nextmove.message;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class BugFix610 {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1118)